Modules/@babylonjs/react-native-iosandroid/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@babylonjs/react-native-iosandroid",
+  "title": "React Native Babylon for iOS and Android",
+  "version": "0.0.1",
+  "description": "Babylon Native integration into React Native",
+  "main": "index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/babylonjs/BabylonReactNative.git",
+    "baseUrl": "https://github.com/babylonjs/BabylonReactNative"
+  },
+  "keywords": [
+    "react-native"
+  ],
+  "author": {
+    "name": "Your Name",
+    "email": "yourname@email.com"
+  },
+  "homepage": "https://github.com/BabylonJS/BabylonReactNative#readme",
+  "license": "MIT",
+  "licenseFilename": "LICENSE",
+  "readmeFilename": "README.md",
+  "dependencies": {
+    "base-64": "^0.1.0",
+    "semver": "^7.3.2"
+  },
+  "peerDependencies": {
+    "@babylonjs/core": "^5.0.0-rc.4",
+    "@babylonjs/react-native": "version",
```

Comment:
Doesn't having this be "version" mess up the local build? Should this be 0.0.1?

Reply:
If it messes up the local build, then PR workflow would trigger an error when building the PG. Or am I missing smth?